src/sentry/snuba/sessions.py

```diff
@@ -173,9 +173,6 @@ def _get_project_releases_by_stability(
         "users": ["-users"],
     }[scope]
 
-    # Tiebreaker
-    orderby.extend(["-project_id", "-release"])
```

Comment:
Would it make sense to leave `order by project_id` intact as a partial tiebreaker?

Reply:
I agree with this

Reply:
oh that makes sense i guess